metrics/integration.go

```diff
@@ -125,15 +125,6 @@ func RegisterCompletedSnapshot(conditiontype, reason string, startTime metav1.Ti
 	SnapshotTotal.With(labels).Inc()
 }
 
-func RegisterInvalidSnapshot(conditiontype, reason string) {
-	SnapshotConcurrentTotal.Dec()
-	SnapshotInvalidTotal.With(prometheus.Labels{"reason": reason}).Inc()
-	SnapshotTotal.With(prometheus.Labels{
```

Comment:
We cannot remove this, this one is valid metric we have to keep it. We want to only remove `SnapshotInvalidTotal` and keep `SnapshotConcurrentTotal` and `SnapshotTotal` in the current state

Reply:
done